docs/chain_documentation/approach.md

```diff
@@ -1,18 +1,27 @@
 # Approach
 
-The Geoclimate library is a collection of spatial processes (algorithms) that allows to compute indicators as form factor, densities, sky view factor... based on vector GIS layers. 
-Each indicator is computed and collected for a specific spatial unit. 
-Indicators are then used in statistical classifications to categorize and extract shared properties or organizations for a studied area. 
+The Geoclimate library is a collection of spatial processes (algorithms) that allows to compute indicators such as form factor, densities, sky view factor... based on vector GIS layers. 
 
-The vector GIS layers used by the algorithms must feed an input data model. 
+Computed for a specific spatial unit, each indicators are then used in statistical classifications to categorize and extract shared properties or organizations for a studied area.
 
-This model is organized around a set of thematic layers: building, road, rail, hydrography, vegetation. 
-
-Each layer follows a dictionary of values that define the name and the type of columns and their constraints 
-(min, max value, allowed string value to describe a feature ...). 
+To do so, the vector GIS layers used by the algorithms must feed an input data model. This model is organized around a set of thematic layers (building, road, railway, hydrography, vegetation)  which must follow a dictionary of values defining the name and type of columns as well as their constraints (min / max value, allowed string value to describe a feature, ...). 
 
 The input data model guarantees the structure of the data sent to the Geoclimate processes while the input data formatting is performed to correct input values that would overpass the defined constraints. 
 
+This approach can be summarized through the main bricks illustrated in the diagram below.
+
+
+
+![approach](../resources/images/chain_documentation/approach.png)
+
+
+
+Remark: The process has been made so that the steps follow one another, but it is also possible to use the bricks *à la carte* according to the data already available to the user and his needs.
```

Comment:
The schema is very nice.
The Remark sentence must be added not as only a remark because the geoclimate is not a workflow but a collection of proceses that can be chained in a workflow.

Reply:
ok